tools/test-hco-utils/test-hco-utils.go

```diff
@@ -0,0 +1,243 @@
+package main
```

Comment:
This should be written as a BDD-style tests using ginkgo to match the other existing functional style tests. See  https://github.com/onsi/ginkgo
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/pkg/controller/hyperconverged/hyperconverged_controller_test.go
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/tests/func-tests/virtual_machines_test.go

Reply:
I need to pass parameters to the test; currently the test gets two command line parameters: 1) the path of the clusterservice yaml file 2) the command name of the kubectl command (so as not to hardcode it). I am still learning ginko, so my questions is: how should I pass these values to the ginko test?

Reply:
I have a problem with ginkgo: the test executable is compiled in the build docker, but copied from the docker to the shell where upgrade-test.sh is running, that is outside of the docker. Here we don't have go and don't have ginkgo so I can't invoke the test other than running the test executable (can't run go test or ginkgo as mentioned in https://onsi.github.io/ginkgo/ )

Reply:
done, now using ginkgo